wolfcrypt/src/random.c

```diff
@@ -284,28 +284,13 @@ enum {
     drbgInitV
 };
 
-/* NOTE: if DRBG struct is changed please update random.h drbg_data size */
-typedef struct DRBG {
-    word32 reseedCtr;
-    word32 lastBlock;
-    byte V[DRBG_SEED_LEN];
-    byte C[DRBG_SEED_LEN];
-#if defined(WOLFSSL_ASYNC_CRYPT) || defined(WOLF_CRYPTO_CB)
-    void* heap;
-    int devId;
-#endif
-    byte   matchCount;
-#ifdef WOLFSSL_SMALL_STACK_CACHE
-    wc_Sha256 sha256;
-#endif
-} DRBG;
-
+typedef struct DRBG_internal __DRBG;
```

Comment:
Is there a good reason to use leading underscores here?  Some compilers may treat that as reserved.

Reply:
Nope.  I just ran s/__DRBG/DRBG_internal/g across random.c to good effect.